extensions/mailer/runtime/src/main/java/io/quarkus/mailer/runtime/MailConfig.java

```diff
@@ -171,6 +182,13 @@ public class MailConfig {
     @ConfigItem(defaultValue = "false")
     public boolean multiPartOnly;
 
+    /**
+     * Caches data from attachment's Stream to a temporary file.
+     * It tries to delete it after sending email.
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean cacheAttachments;
```

Comment:
This doesn't seem to belong to this PR?

Reply:
It affects the DKIM signing performance, see [here](https://vertx.io/docs/4.1.0/vertx-mail-client/java/#_dkim_signature_signing_emails)

Reply:
But can you point me to where is it being used?

Reply:
sure. https://github.com/quarkusio/quarkus/pull/25248/files#diff-f29996b22d408f02c02fac60e7c68440ad754e730f505c42836017d3b3fcdb3bR61

Reply:
Lol, completely missed that. Thanks. I think it needs to be produced through a build item?

Reply:
What do you mean by build item? Sorry for the question, still learning.

Reply:
Ok, got it. checking how to do it.

Reply:
I think you need to produce a `SystemPropertyBuildItem`. Here is an example: 

https://github.com/quarkusio/quarkus/blob/64173d81a8d9bb2b546481c09d480d5ff3830e0b/extensions/liquibase/deployment/src/main/java/io/quarkus/liquibase/deployment/LiquibaseProcessor.java#L85-L91